integration_test/third_party_apps_test.go

```diff
@@ -18,12 +18,14 @@ The following variables are optional:
 MONITORING_AGENT_SUFFIX: If provided, repo suffix to provide when installing the monitoring agent.
 AGENT_PACKAGES_IN_GCS: If provided, a URL for a directory in GCS containing
     .deb/.rpm/.goo files to install on the testing VMs. They must be inside
-	a directory called ops-agent. For example, this would be a valid structure
-	inside AGENT_PACKAGES_IN_GCS:
+    a directory called ops-agent. For example, this would be a valid structure
+    inside AGENT_PACKAGES_IN_GCS:
     └── ops-agent
         ├── ops-agent-google-cloud-1.2.3.deb
         ├── ops-agent-google-cloud-1.2.3.rpm
         └── ops-agent-google-cloud-1.2.3.goo
+REPO_SUFFIX: If provided, a Rapture repository suffix to install the agent from.
```

Comment:
We try to avoid using "Rapture" externally. How about just "a package repository suffix…"?

Reply:
done